src/views/news.hbs

```diff
@@ -0,0 +1,18 @@
+<div>
+  <div>
+    <span>{{this.news.date}}</span>
+    <a href="linkedin.com"><img src="" alt=""></a>
+    <a href="facebook.com"><img src="" alt=""></a>
+    <a href="twitter.com"><img src="" alt=""></a>
+  </div>
+
```

Comment:
They are for sharing the article on facebook, @Karyum 
But I think we can leave them out for now, as you say @Suuuuha , because the PO might not go ahead with social media anyway.